test/typescript/exposed.test.ts

```diff
@@ -14,4 +14,6 @@ const mockWithT: WithT = {
 const modules: Modules = { external: [] };
 
 const resourceStore: ResourceStore = i18next.services.resourceStore;
+resourceStore.on('added', console.log);
+resourceStore.off('added', console.log);
```

Comment:
Please add another usage test for the optional listerner, e.g. `resourceStore.off('added')`

Reply:
@jokester as soon as you've completed the suggestions of @rosskevin we will merge this

Reply:
@rosskevin @adrai  Sorry I didn't see the notifications. Thanks for adding the missing code. Are there more to add?